tests/st/runtime/test_mat_slice_to_left.py

```
@@ -0,0 +1,142 @@
+# See LICENSE in the root of the software repository for the full text of the License.
+# -----------------------------------------------------------------------------------------------------------
+
+"""
+Tests for Mat slice → Left/Right matmul pattern (issue #1198).
+
+Validates the pattern where a wide BF16 tile is loaded into Mat (L1) to satisfy
+the >= 512 B GM row-alignment constraint, then sliced into two K-chunks that are
+moved to Left for matmul.  This pattern appears in Qwen3 decode kernels (q_proj,
+kv_proj, out_proj, gate_proj, up_proj, down_proj) where K_CHUNK=128 with BF16
+yields only 256 B per row — below the 512 B minimum.  Merging two adjacent
+K-chunks into one wider GM load (row = 2 × 128 × 2 = 512 B) and splitting
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)